src/app/components/publiclists/challengelist/challengelist.component.ts

```diff
@@ -240,6 +261,9 @@ export class ChallengelistComponent implements OnInit {
   filterChallengesByTeams() {
     if (this.router.url === this.myChallengesRoutePathCommon && this.authService.isLoggedIn()) {
       this.filteredChallenges = this.filteredChallenges.filter((v, i, a) => this.allTeams.indexOf(v['creator']['id']) > -1);
+      this.filteredOngoingChallenges = this.filteredOngoingChallenges.filter((v, i, a) => this.allTeams.indexOf(v['creator']['id']) > -1);
```

Comment:
May be there is better way of filtering - means only one iteration. It is fine for now.